webapp/src/main/java/com/box/l10n/mojito/service/ai/openai/OpenAILLMService.java

```diff
@@ -56,6 +63,23 @@ public class OpenAILLMService implements LLMService {
   @Value("${l10n.ai.checks.persistResults:true}")
   boolean persistResults;
 
+  @Value("${l10n.ai.translate.retry.maxAttempts:10}")
+  int retryMaxAttempts;
+
+  @Value("${l10n.ai.translate.retry.minDurationSeconds:5}")
+  int retryMinDurationSeconds;
+
+  @Value("${l10n.ai.translate.retry.maxBackoffDurationSeconds:60}")
+  int retryMaxBackoffDurationSeconds;
+
+  @Value("${l10n.ai.translate.response.isJson:false}")
+  boolean isTranslateResponseJson;
```

Comment:
When we expect JSON responses - are we also instructing the client to ask for a JSON response in the API request call, using either structured outputs `{ "type": "json_schema", "json_schema": {...} } ` or JSON mode `{ "type": "json_object" }`? (In addition the the contents of the prompt).

That would be information that would to be stored alongside the prompt in the DB.

The question here would be: what is the impact of requesting (in the system/user prompt) for a JSON response, without also hinting re the format in the API call parameters and if that's something that we want to do?

Reply:
In my local tests I have not been setting the json mode and the model has successfully returned a valid json object. Will update the prompt table to allow us to store this setting and then set that in the request accordingly.

Anecdotally though having json mode disabled and requesting a json response has worked in my limited testing so far against the API.